pubspec.lock

```diff
@@ -229,10 +236,10 @@ packages:
   scanbot_sdk:
     dependency: "direct main"
     description:
-      name: scanbot_sdk
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "2.2.2"
+      path: "../scanbot-sdk-flutter"
```

Comment:
The public example shouldn't use a local SDK version

Reply:
I can suggest that this is for testing purposes. We can't merge this until the release, as far I understand.